cmd/plugin.go

```diff
@@ -201,6 +205,10 @@ func runPluginInstallCmd(cmd *cobra.Command, args []string) {
 	// plugin names can be simple names ('aws') for "standard" plugins,
 	// or full refs to the OCI image (us-docker.pkg.dev/steampipe/plugin/turbot/aws:1.0.0)
 	plugins := append([]string{}, args...)
+	silent, err := cmd.Flags().GetBool("silent")
```

Comment:
Is there any reason you are pulling the value from the cmdFlags as opposed to `viper` like we do in other places?

Reply:
To put this in context, all flag values are extracted and put in a global key value map. This is done in the PreRun binding phase.

Example:
https://github.com/turbot/steampipe/blob/f47985f88fc111d267640afec062e67f09d62efa/cmd/check.go#L143

Reply:
I am not sure yet, but I suspect this is why the short hand is not working